.github/workflows/ci-pip-workflow.yml

```diff
@@ -92,21 +92,22 @@ jobs:
           NUMPYVER='numpy'
         fi
         pip install ${NUMPYVER} ${FITSBUILD} ${MATPLOTLIBVER} ${BOKEHVER} ${ARVIZVER}
+        pip install pytest-xvfb
```

Comment:
If this is going to be moved here then why not do it in line 94?

Reply:
Either works. I thought it's easier to read to have one line with all the variables and one line with the hardcoded names (currently only one), but I'm fine with putting it on the same line, too, if other people prefer that.
Since pip doesn't do a long dependency resolution like conda does, there shouldn't be a noticeable effect on runtime.